README.md

```diff
@@ -21,36 +21,36 @@ Or:
 
     pyp2rpm package_name --srpm
 
-This downloads the package from PyPI and creates SRPM file.
+This downloads the package from PyPI and creates ab SRPM file.
```

Comment:
I fixed that [here ](https://github.com/fedora-python/pyp2rpm/commit/6054c0a7e3029fac1b9e63f7cabcf593a40ace3f#diff-04c6e90faac2675aa89e2176d2eec7d8R23) before merge .